plugins/MauticCrmBundle/Integration/PipedriveIntegration.php

```diff
@@ -111,75 +111,81 @@ public function isAuthorized()
     }
 
     /**
-     * Get available company fields for choices in the config UI.
-     *
-     * @param array $settings
-     *
-     * @return array
-     */
-    public function getFormCompanyFields($settings = [])
```

Comment:
Removing or renaming a public or private method is a BC break. If you want to get rid of it, mark it as `@deprecated in 2.x.x, to be removed in 3.0.0, use xxx instead` in the annotation. You can edit its content to use the new method too. But it should return data in the same format as before.

Reply:
You, right getFormCompanyFields will be back.

Reply:
Just question. Is getFormCompanyFields required if the parent abstract class already contains and works properly

https://github.com/kuzmany/mautic/blob/ef4ba891e182c33bd65c9e8263c61a7e6f6bad9c/app/bundles/PluginBundle/Integration/AbstractIntegration.php#L2455-L2455